app/api/resources/user.py

```diff
@@ -57,8 +57,12 @@ def get(cls):
         location, occupation, organization, interests, skills, need_mentoring,
         available_to_mentor. The current user's details are not returned.
         """
+
+        page = request.args.get("page", default=UserDAO.DEFAULT_PAGE, type=int)
+        per_page = request.args.get("per_page", default=UserDAO.DEFAULT_USERS_PER_PAGE, type=int)
+
```

Comment:
Here check that page and per_page is integer or not. According to this raise error.
You are using page and per_page as integer, if possible update the swagger as well currently it is showing string.

Reply:
@satya7289 I searched a bit documentation, and how it is now I am not sure if we can change the documentation with the parameter type. So could we leave this for later investigation? This seems like a documentation problem only, and not functional

https://flask-restplus.readthedocs.io/en/stable/swagger.html#method-parameters

cc @lmcicat

Reply:
If I put abc(string) in page parameter or per_page parameter it should raise parameter error. In the current scenario I guess it is taking default values if we pass string value to page or per_page parameter.

Reply:
> I guess it is taking default values if we pass string value to page or per_page parameter.

if this is the case then this can pass in this PR, and improved later right? Not only improved but discussed. Because maybe we want to ignore instead of raising an error. In my opinion that would be fine. @satya7289 could we leave this for further discussion later?

Reply:
Ok